mockld/events_service.go

```diff
@@ -0,0 +1,79 @@
+package mockld
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/launchdarkly/sdk-test-harness/framework"
+)
+
+// EventsService is a simulation of the LaunchDarkly event-recorder service, allowing tests to
+// receive event data from an SDK.
+type EventsService struct {
+	AnalyticsEventPayloads chan Events
+	sdkKind                SDKKind
+	credential             string
+	logger                 framework.Logger
+}
+
+func NewEventsService(sdkKind SDKKind, credential string, logger framework.Logger) *EventsService {
+	return &EventsService{
+		AnalyticsEventPayloads: make(chan Events, 100),
```

Comment:
Is 100 arbitrary?

Reply:
Yes.

Reply:
The intention is just to make it unlikely for us to get blocked by a channel being full. The goroutine that listens for event posts should be able to just dump the data onto the channel and call it a day; there's no value in this case in having it block till we've read the value from the channel.

Reply:
I think it'd be worth encoding this answer into a comment on a constant written in the code.